snap/squashfs/squashfs.go

```diff
@@ -90,6 +90,14 @@ func (s *Snap) Install(targetPath, mountDir string) error {
 
 var runCommandWithOutput = func(args ...string) ([]byte, error) {
```

Comment:
I slightly disagree, perhaps this could be just called `runCommandWithEnglishOutput`, there are no side-effects (the change affects the child and not the caller), unlike in the global test proposal.